docs/changelog/125060.yaml

```diff
@@ -0,0 +1,16 @@
+pr: 125060
+summary: Allow partial results by default in ES|QL
+area: ES|QL
+type: breaking
+issues: []
```

Comment:
Do we want maybe put https://github.com/elastic/elasticsearch/issues/122802 here?

Reply:
Sure, https://github.com/elastic/elasticsearch/pull/125060/commits/5ffc0d71cadf28128f483adcbdb0609f534e053d